core-services/src/main/java/org/zalando/nakadi/service/SchemaService.java

```diff
@@ -288,9 +293,10 @@ public InputStream get(final String ref) throws SchemaValidationException {
         }
     }
 
-    public static void isStrictlyValidJson(final String jsonInString) throws InvalidEventTypeException {
+    @VisibleForTesting
+    public static JSONObject parseJsonSchema(final String jsonInString) throws InvalidEventTypeException {
```

Comment:
```suggestion
    private static JSONObject parseJsonSchema(final String jsonInString) throws InvalidEventTypeException {
```
Did you mean to make it private?

Reply:
> 1. Did you mean to make it private?

This is still called in tests. At most I have changed it to package private (otherwise, tests needs to be rewritten too).


> 2. Looks like `throws` part is outdated.

Good point!